release/platform/platform.go

```diff
@@ -215,32 +222,49 @@ var platforms = []Platform{
 		Repos: []string{RepoOrg, RepoEnterprise},
 	},
 	{
-		Name:  "rhel71",
-		Arch:  "ppc64le",
+		Name:  "ubuntu2004",
+		Arch:  "arm64",
 		OS:    OSLinux,
-		Pkg:   PkgRPM,
-		Repos: []string{RepoEnterprise},
+		Pkg:   PkgDeb,
+		Repos: []string{RepoOrg, RepoEnterprise},
 	},
 	{
-		Name:  "ubuntu1604",
+		Name: "rhel71",
+		Arch: "ppc64le",
+		OS:   OSLinux,
+		Pkg:  PkgRPM,
+	},
+	{
+		Name: "rhel81",
+		Arch: "ppc64le",
+		OS:   OSLinux,
+		Pkg:  PkgRPM,
+	},
+	{
+		Name: "ubuntu1604",
+		Arch: "ppc64le",
+		OS:   OSLinux,
+		Pkg:  PkgDeb,
+	},
+	{
+		Name:  "ubuntu1804",
 		Arch:  "ppc64le",
 		OS:    OSLinux,
 		Pkg:   PkgDeb,
 		Repos: []string{RepoOrg, RepoEnterprise},
 	},
 	{
-		Name:  "ubuntu1804",
+		Name:  "ubuntu2004",
 		Arch:  "ppc64le",
 		OS:    OSLinux,
 		Pkg:   PkgDeb,
 		Repos: []string{RepoOrg, RepoEnterprise},
 	},
 	{
-		Name:  "rhel67",
-		Arch:  "s390x",
-		OS:    OSLinux,
-		Pkg:   PkgRPM,
-		Repos: []string{RepoOrg, RepoEnterprise},
+		Name: "rhel67",
+		Arch: "s390x",
+		OS:   OSLinux,
+		Pkg:  PkgRPM,
```

Comment:
why was the "repos" key removed from this platform and multiple others?

Reply:
Hmm, I didn't notice this at first so good catch! Most of the changes to this file were from when I reverted one of the commits mentioned in the ticket, so something might've gotten lost in translation there on my end. I'll double-check everything to make sure it's as-expected.